release_tester/selenium_ui_test/pages/analyzers_page.py

```diff
@@ -680,7 +679,7 @@ def add_new_analyzer(self, name, ui_data_dir=None):
             # ----------------------adding first pipeline analyzer as Norm analyzer--------------------------
             self.tprint(f'Selecting add analyzer button for {name} \n')
             if self.version_is_newer_than('3.11.99'):
-                add_analyzer01 = '//*[@id="chakra-modal--body-7"]/div/div[3]/div/button'
+                add_analyzer01 = "(//button[@class='chakra-button css-oni4cz'])[1]"
```

Comment:
this `css-oni4cz` can change at any time. Let's add test IDs and use those here

Reply:
Taken care of.